CONTRIBUTORS.md

```diff
@@ -6,21 +6,30 @@ Committers
 ----------
 Committers are people who have made substantial contribution to the project and granted write access to the project.
 * [Tianqi Chen](https://github.com/tqchen), University of Washington
-  - Tianqi is a PhD working on large-scale machine learning, he is the creator of the project.
+  - Tianqi is a Ph.D. student working on large-scale machine learning. He is the creator of the project.
 * [Tong He](https://github.com/hetong007), Amazon AI
-  - Tong is an applied scientist in Amazon AI, he is the maintainer of xgboost R package.
+  - Tong is an applied scientist in Amazon AI. He is the maintainer of XGBoost R package.
 * [Vadim Khotilovich](https://github.com/khotilov)
   - Vadim contributes many improvements in R and core packages.
 * [Bing Xu](https://github.com/antinucleon)
-  - Bing is the original creator of xgboost python package and currently the maintainer of [XGBoost.jl](https://github.com/antinucleon/XGBoost.jl).
+  - Bing is the original creator of XGBoost Python package and currently the maintainer of [XGBoost.jl](https://github.com/antinucleon/XGBoost.jl).
 * [Michael Benesty](https://github.com/pommedeterresautee)
-  - Micheal is a lawyer, data scientist in France, he is the creator of xgboost interactive analysis module in R.
+  - Micheal is a lawyer and data scientist in France. He is the creator of XGBoost interactive analysis module in R.
 * [Yuan Tang](https://github.com/terrytangyuan)
   - Yuan is a data scientist in Chicago, US. He contributed mostly in R and Python packages.
```

Comment:
Could you update my information to: " a software engineer in Ant Financial"? Thanks.